test/model/test_forecast.py

```diff
@@ -75,6 +75,23 @@ def percentile(value):
     forecast.plot()
 
 
+def test_mean_only_forecast():
+    forecast = QuantileForecast(
+        forecast_arrays=np.ones(shape=(1, 12)),
+        start_date=pd.Period("2022-03-04 00", freq="H"),
+        forecast_keys=["mean"],
+    )
+
+    assert forecast.prediction_length == 12
+    assert len(forecast.index) == 12
+    assert forecast.index[0] == pd.Period("2022-03-04 00", freq="H")
+
+    for level in [0.1, 0.5, 0.7]:
```

Comment:
Ok. It seems that the main message here is that, if we only have the field `mean` in the `QuantileForecast` then we will return `np.nan` if any quantile is requested. LGTM.

Reply:
Exactly